sklearn/utils/tests/test_class_weight.py

```diff
@@ -26,6 +29,27 @@ def test_compute_class_weight_not_present():
     assert_raises(ValueError, compute_class_weight, "auto", classes, y)
 
 
+def test_compute_class_weight_invariance():
```

Comment:
Yes. Thanks for checking. I guess I should put more of your text in the comment so the next reader (aka future me) has it easier.